src/test/scala/com/wavesplatform/state/diffs/ExchangeTransactionDiffTest.scala

```diff
@@ -466,14 +466,14 @@ class ExchangeTransactionDiffTest extends PropSpec with PropertyChecks with Matc
         .selfSigned(1: Byte, buyer, buyerScript, enoughFee, ts + 7)
         .explicitGet()
       assetPair = AssetPair(Some(asset1.id()), Some(asset2.id()))
-      o1 <- Gen.oneOf(
-        OrderV1.buy(seller, MATCHER, assetPair, 1000000, 1000000, ts + 8, ts + 10000, enoughFee),
-        OrderV2.buy(seller, MATCHER, assetPair, 1000000, 1000000, ts + 8, ts + 10000, enoughFee)
-      )
-      o2 <- Gen.oneOf(
-        OrderV1.sell(buyer, MATCHER, assetPair, 1000000, 1000000, ts + 9, ts + 10000, enoughFee),
-        OrderV2.sell(buyer, MATCHER, assetPair, 1000000, 1000000, ts + 9, ts + 10000, enoughFee)
-      )
+      o1        = //Gen.oneOf(
```

Comment:
shouldn't we check OrderV1 in tests anymore?

Reply:
OrderV1 is tested now.